src/bio/bio_internal.h

```diff
@@ -60,15 +60,15 @@ struct bio_dma_buffer {
 enum bio_bs_state {
 	/* Healthy and fully functional */
 	BIO_BS_STATE_NORMAL	= 0,
-	/* Being marked as faulty by monitor, rebuilding pool targets */
+	/* Being detected as faulty */
 	BIO_BS_STATE_FAULTY,
-	/* Rebuild triggered, tearing down SPDK resources */
+	/* Affected targets are marked as DOWN, safe to tear down blobstore */
 	BIO_BS_STATE_TEARDOWN,
-	/* SPDK blobs, blobstore and bdev are torn down */
+	/* Blobstore is torn down */
 	BIO_BS_STATE_OUT,
-	/* New device hotplugged, initializing SPDK resources */
+	/* New device hotplugged, start to initialize blobstore & blobs */
```

Comment:
maybe add comment about how this state is not currently supported?

Reply:
Sure, will improve the comment it in follow-on patch.